src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -238,23 +251,27 @@
 <script lang="ts">
 import { Component, Emit, Mixins, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'pinia-class'
+import { mask } from 'vue-the-mask'
 import { useStore } from '@/store/store'
 import { DateMixin } from '@/mixins'
-import { cloneDeep, isEqual } from 'lodash'
-import { EmptyBusinessLookup, EmptyExistingBusinessInfoIF, ExistingBusinessInfoIF } from '@/interfaces'
+import { ExistingBusinessInfoIF } from '@/interfaces'
+import { Rules } from '@/rules'
+import { VuetifyRuleFunction } from '@/types'
 import MessageBox from '@/components/common/MessageBox.vue'
 import { Jurisdiction } from '@bcrs-shared-components/jurisdiction'
 import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker'
 import { JurisdictionLocation } from '@bcrs-shared-components/enums'
 import { CountriesProvincesMixin } from '@/mixins/'
 import { FormIF } from '@bcrs-shared-components/interfaces'
-import { MrasJurisdictions } from '@bcrs-shared-components/jurisdiction/list-data'
 
 @Component({
   components: {
     DatePickerShared,
     Jurisdiction,
     MessageBox
+  },
+  directives: {
+    mask
```

Comment:
First time I've seen this, nice! The import reminds me of the movie haha

I don't see it being used though? Or is it related to the v-mask in the template?

Reply:
Haha, me too.

Yes, this is the v-mask directive in the template.

Ref: https://www.npmjs.com/package/v-mask